aws/emr/cdo-logs/script/elb_mapper.rb

```diff
@@ -1,13 +1,103 @@
 #!/usr/bin/ruby
 
+# The list of tutorials. Note that relative order matters, e.g., it is important
+# that "starwarsblocks" proceeds "starwars" as the latter will match the regex
+# for the former.
+tutorials = [
+    "mc",
+    "starwarsblocks",
+    "starwars",
+    "tynker",
+    "frozen",
+    "flappy",
+    "hourofcode",
+    "codecombat",
+    "tynkerapp",
+    "lightbotintl",
+    "lightbot2",
+    "lightbot",
+    "scratch",
+    "khanes",
+    "khanfr",
+    "khanhe",
+    "khanpl",
+    "khanpt",
+    "khan",
+    "playlab",
+    "makegameswithus",
+    "touchdevelop",
+    "codemonkey",
+    "codecademy",
+    "processing",
+    "codespark",
+    "groklearning",
+    "appinventor",
+    "codeavengers",
+    "codehs",
+    "bitsbox",
+    "agentcubes",
+    "thinkersmith",
+    "gumball",
+    "artist",
+    "blockly",
+    "kodable",
+    "infinity",
+    "robomind",
+    "iceage",
+    "codeintl",
+    "allcancode",
+    "hopscotch",
+    "makeschool",
+    "thinkersmithspanish",
+    "condcards",
+    "monstercoding",
+    "thinkersmith2",
+    "kodableunplugged",
+    "lookingglass",
+    "russia_mc",
+    "csfirst",
+    "livecode",
+    "alice",
+    "quorum",
+    "projguts",
+    "codesters",
+    "boxisland",
+    "teacherled",
+    "kodableapp",
+    "texasinstruments",
+    "robomindnl",
+    "baymaxes",
+    "pixies",
+    "finchrobot",
+    "disney_static_starwars",
```

Comment:
should this (and the two other tutorials that include "starwars") also precede the "starwars" line above? Similarly for "static_mc"?